.github/CONTRIBUTING.md

```diff
@@ -61,6 +61,16 @@ If the code is mergeable (you get a message saying “Able to merge”), go ahea
     - Check back after some time to see if the Travis checks have passed, if not you should click on “Details” link on your PR thread at the right of “The Travis CI build failed”, which will take you to the dashboard for your PR. You will see what failed / stalled, and will need to resolve them.
     - If your checks have passed, your PR will be assigned a reviewer who will review your code and provide comments. Please address each review comment by pushing new commits to the same branch (the PR will automatically update, so you don’t need to submit a new one). Once you are done, comment below each review comment marking it as “Done”. Feel free to use the thread to have a discussion about comments that you don’t understand completely or don’t agree with.
 
-    - Once all comments are addressed, the reviewer will give an LGTM (‘looks good to me’) and merge the PR.
+    - Once all comments are addressed, the reviewer will give an LGTM (‘looks good to me’)
+
+4. Once you get reviewed by a mentor and done with all the required changes, squash all the commits:
```

Comment:
@Kajol-Kumari Can you please add a GitHub document link for reference.

Reply:
@RishabhJain2018 the github documentation is only available for master as here https://github.com/wprig/wprig/wiki/How-to-squash-commits. This may confuse the newbies so i think https://www.internalpointers.com/post/squash-commits-into-one-git will give better clarification if required.

Reply:
Done @RishabhJain2018